api/v1alpha1/dspipeline_types.go

```diff
@@ -44,6 +44,12 @@ type DSPASpec struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default:="v1"
 	DSPVersion string `json:"dspVersion,omitempty"`
+
+	// PodToPodTLS Set to "true" or "false" tls communication between DSPA components. Default to enable Inter-Pod tls by setting to "true". Only supported in DSP V2 and OpenShift.
```

Comment:
```suggestion
	// PodToPodTLS Set to "true" or to enable TLS communication between DSPA components (pods), "false" to disable it. In most environments, it's best to enable this, so it defaults to "true". Only supported in DSP V2.
```

Do we need to mention OpenShift here? DSPO only works on OpenShift right?

Reply:
technically no, we run it in kind cluster just fine for testing